packages/spor-react/src/theme/slot-recipes/breadcrumb.ts

```diff
@@ -0,0 +1,54 @@
+import { coreBackground, coreBorder } from "../utils/core-utils";
+import { ghostBackground } from "../utils/ghost-utils";
+import { defineSlotRecipe } from "@chakra-ui/react";
+import { breadcrumbAnatomy } from "./anatomy";
+import { focusVisibleStyles } from "../utils/focus-utils";
+
+export const breadcrumbSlotRecipe = defineSlotRecipe({
+  className: "spor-breadcrumb",
+  slots: breadcrumbAnatomy.keys(),
+  base: {
+    list: {
+      display: "inline-flex",
+      flexWrap: "nowrap",
+      alignItems: "center",
+    },
+    link: {
+      cursor: "pointer",
+      padding: "3px",
+      borderRadius: "6px",
+      _focusVisible: {
+        ...focusVisibleStyles()._focusVisible,
+      },
```

Comment:
Eventuelt kan du bare skrive
```suggestion
     ...focusVisibleStyles(),
```